File: leetcode-submissions/1528_Shuffle_String.cpp
class Solution {
public:
    string restoreString(string s, vector<int>& indices) {
        unordered_multimap<char,int>mp;
        for(int i=0;i<s.size();i++){
            mp.insert(make_pair(s[i],indices[i]));

        }
        vector<char>temp(s.size());
        for(auto ele:mp){
            temp[ele.second]=ele.first;
        }

        string ans(temp.begin(),temp.end());\
        return ans;

    }
};

File: leetcode-submissions/1528_Shuffle_String.ts
function restoreString(s: string, indices: number[]): string {
    const n:number = indices.length;
    const arr:string[]= new Array(n);
    for(let i:number = 0 ;i<n;i++){
        arr[indices[i]]= s[i]; 
    }
    return arr.join("");
};